asv_bench/benchmarks/reshape.py

```diff
@@ -146,4 +146,26 @@ def time_get_dummies_1d_sparse(self):
         pd.get_dummies(self.s, sparse=True)
 
 
+class Cut(object):
+    params = [[4, 10, 1000]]
+    param_names = ['bins']
+
+    def setup(self, bins):
```

Comment:
Could you also add datetime and timedelta data to this cut benchmark? After that LGTM

Reply:
Done